docs/Config_Reference.md

```diff
@@ -2519,6 +2519,21 @@ serial_no:
 #   The micro-controller to read from. Must be the host_mcu
 ```
 
+### Combined temperature sensor
+
+Combined temperature sensor is a virtual temperature sensor based on several other sensors. This sensor can be used with extruders and heater beds.
+
```

Comment:
If this works with `[heater_generic]` sections too, that would be a good item to add.

Reply:
Fixed via https://github.com/Klipper3d/klipper/commit/501394345675eb30dcd66b055d386dc34ca4e27b